perf/tree_gen/script.js

```diff
@@ -0,0 +1,120 @@
+import * as gen from '../../src/world/tree_generation.js';
+
+
+const DEFAULT_CONFIG = {
+  wSize: [50, 12, 50], 
+  nTrees: 170, 
+  treeRadius: [1, 1],
+};
+
+
+export function run(conf=DEFAULT_CONFIG) {
+  var suite = Benchmark.Suite("tree placers");
+  
+  function getPlacerFn(cls) {
+    return () => {
+      new cls({
+        cnf: {
+          generation: {
+            ...conf,
+            seed: Math.random()
+          }
+        }
+      }).makeTrees()
+    }
+  }
+  
+  function addCls(cls) {
+    suite.add(cls.name, getPlacerFn(cls));
+  }
+  
+  addCls(gen.IgnoreTreePlacer);
+  addCls(gen.SkipTreePlacer);
+  addCls(gen.AvoidTreePlacer);
+  addCls(gen.AvoidTreePlacerFast);
+
+  globalThis.perfSuite = suite;
+  runSuite(suite);
+}
+
+
+export function makeElem(tagname, children=[], classList=[]) {
+  let e = document.createElement(tagname);
+  e.append(...children);
+  e.classList.add(...classList);
+  return e;
+}
+
+export function runSuite(suite) {
```

Comment:
This `runSuite` function seems quite useful, we should extract it into its own module: `perf/lib.js` or `perf/common.js`?

Reply:
This would make writing future performance tests easier